refactor_me.py

```diff
@@ -2,25 +2,49 @@
 
 from collections import namedtuple, defaultdict
 from operator import itemgetter
+from sys import stdout
+import io
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', ('category', 'amount'))
 
 
 def sum_expenses(expenses, min_amount=0):
+    """Aggregates all Expense entries into a categorized list of sums
```

Comment:
I would add a doc-string at all since it does not provide extra information that is already provided by function and argument names and the code. The only reason I see to add doc-string for this function if we generate documentation meant to be read without the code.

Reply:
In most cases, I wouldn't add a doc-string for such a small function that performs such a simple task. What they do provide is context for IDEs for debugging and interspection in cases where I can't provide type casting (which would break support for Python <3.5)